main.go

```diff
@@ -13,15 +13,27 @@ import (
 
 var collection string
 var minCheckins int
+var is2010 bool
+var tfs2010Collections = [18]string{"CareSystems", "ClaimsEncounters", "CoreSystems","CorpSystemsNon-Erp", "CRM", "DBA","DefaultCollection","Encounters", "E-portal", "ITOps","Octo","ODSDWBI",
+"PRELEX","Provider-Eligibility-Extracts","QNXT-Reports-Utilities","Security","SOA","TCIM"}
+var rootPath string
 
 func main() {
 	flag.StringVar(&collection, "collection", "http://tfs.molina.mhc:8080/tfs/HSS/", "TFS collection URL")
 	flag.IntVar(&minCheckins, "mincheckins", 10, "minimum number of check-ins for active users")
+	flag.BoolVar(&is2010, "2010", false, "Use the 2010 Repo")
 	flag.Parse()
-
-	h := history()
+	h := make([]string,0)
```

Comment:
This is more idiomatic if you don't know the size 
```go
var h []string
```
See here https://stackoverflow.com/questions/25543520/declare-slice-or-make-slice